### ISSUE TITLE
Incorrect/missing RSS feed links

### ISSUE THREAD

**Issue by ngm** [TheRestartProject/therestartproject.org#85]
The link to the rss feed (<link rel="alternate" type="application/rss+xml") is wrong on the home page and missing on several others.

**Comment by ngm** [comment]
@acrostich for the other pages, where the feed is missing, what would you expect to find here?  Would you expect on all pages to find the RSS feed for the full blog listing?

**Comment by acrostich** [comment]
1) As these days people use search engines to go to a website, you can't expect people to go to the homepage to find the rss feed(s). So I would have the same alternate tags on all pages.

2) The blog posts feed is the essential one, but you could offer others as well such as feed of blog comments, feed of events, etc. This may also list the feed for the podcast (I can't remember how I discovered it initially).

If you do offer several feeds, list them in order of importance (although this is of course subjective).

**Comment by acrostich** [comment]
Yes that would be helpful. In that case this would be the first or second choice on that page (and obviously don't appear on other unrelated pages).

On a post within a category, this should be offered as well.

If a post can have multiple categories, then it would make sense to offer feeds for all the categories it belongs to.

(However if this is too complex, another option would be to have only the blog posts feed in the alternate tag, and then have a page, linked from the bottom, with all the other feeds as these are likely to be of less interest.)

**Comment by ngm** [comment]
## specs

### homepage

This is the main priority.

RSS feeds should be, in this order: 

* Blog posts: https://therestartproject.org/feed/
* Podcast: https://therestartproject.org/podcast/feed

A feed for events could be nice, and does exist at https://therestartproject.org/feed/parties, however could do with some work before advertising it, as it currently only includes publication date (see e.g. http://carp.docs.geckotribe.com/feed-test.php?feed=https%3A%2F%2Ftherestartproject.org%2Fparties%2Ffeed%2F) 

### category pages and posts within categories

e.g. https://therestartproject.org/education 

First the feed for the category, then the full blog feed.

e.g.
* https://therestartproject.org/education/feed/
* https://therestartproject.org/feed/

Note: podcast is a category too.

### all other pages

Same as the homepage.

**Comment by acrostich** [comment]
On August 14th, Ugo wrote: 'To improve our communication on upcoming London events, we are concentrating on Meetup.' If that statement is still valid, for events you can include this feed: webcal://www.meetup.com/Restarters-London/events/ical/

In the 'category pages' section:

I'd suggest you keep the two or three feeds listed under 'homepage' and add the category-specific feed underneath. The consistency is more important than having to click the fourth (or third) link down instead of the fourth for this cases.

And similarly for 'blog posts that existing within a category'.

**Comment by ngm** [comment]
Thanks @acrostich - have updated my comment.

**Comment by PhilipAbbott** [comment]
On the homepage I can correct the current incorrect link for the podcast subscription but where would you like the subscribe to the blog in general? I can add this in to the header as a button with the other social media items, but judging by the conversation you may want to make it more obvious on the main page. The problem we have here is that there is not a section on the homepage that list the blog posts as we have them split into community and education blogs. I can add buttons to subscribe to those two separate blog feeds easily but then where would I put the subscribe to the blog in general button. I have mocked up a suggestion below:

![homepage-rss](https://user-images.githubusercontent.com/5479226/28501333-b20fa88e-6fd1-11e7-8ddb-fe6aeddb5a49.png)

On the other pages I would also like a guide on where to put the list of blogs to subscribe to. Either I can add them to the sidebar or the footer. An indication of where on the page you would like the element would be helpful.

**Comment by acrostich** [comment]
Browsers (should) have discovery mechanism for <link rel="alternate" type="application/rss+xml" so using the browser's own mechanism should be enough. 

The feeds for the blog categories (Education and Community) don't need to be discovered on the home page, but only when in these respective categories I find. The home page already has a lot of content.

If you want to make it clear outside of the browser discovery mechanism what feeds are available I'd have a page listing all the feeds and linked from the very bottom.

**Comment by acrostich** [comment]
BTW there's another feed (to which I subscribe to), that of changes to the wiki: https://therestartproject.org/wiki/index.php?title=Special:RecentChanges&feed=atom

It is a rather specialised feed and I wouldn't include it in the alternate list, however if you do create a page listing all the available feeds then it would make sense to include it there.

**Comment by ngm** [comment]
Hey @PhilipAbbott - sorry, I didn't make it very clear - these are not to be visible on the page, but are included in the `<head>` and then picked up by feed readers.  

So for example if you look in the source e.g. on the homepage at the moment, we have in the head:

```
<link rel="alternate" type="application/rss+xml" title="The Restart Project » Homepage Comments Feed" href="https://therestartproject.org/homepage/feed/">
```

but it should be:

```
<link rel="alternate" type="application/rss+xml" title="The Restart Project - Blog" href="https://therestartproject.org/feed/">
<link rel="alternate" type="application/rss+xml" title="The Restart Project - Podcast" href="https://therestartproject.org/podcast/feed/">
```

I don't know what Wordpress' mechanism is for adding the alternate links - had a very quick glance in `header.php` and didn't see anything directly in there - guessing it is added somewhere in  `wp_head`? 

Once you determine where in WP the links are being built, then the most important thing is simply getting the full blog feed as the alternate link as opposed to that current incorrect one.  But I'd like to also find out how feasible it is to get multiple alternate links per page, and different alternate links based on the current page the visitor in on.

If multiple/conditional alternate links ends up being non-trivial to add, then we will go instead with just the one main alternate link in the head and have a separate page listing all the RSS feeds.

**Comment by PhilipAbbott** [comment]
After a bit of poking around I am fairly sure changing the wordpress functions in the following files would fix this
https://gitlab.com/therestartproject/therestartproject.org/blob/master/wp-includes/feed-rss.php
https://gitlab.com/therestartproject/therestartproject.org/blob/master/wp-includes/feed-rss2.php

These values may be editable from the WP backend, if not I could hard code them in but obviously that's  a less than ideal solution unless they are never likely to change.
